classes/controllers/tab/workflow/PKPReviewRoundTabHandler.inc.php

```diff
@@ -85,6 +85,18 @@ protected function _reviewRound($args, $request) {
 		);
 		$templateMgr->assign('reviewRoundNotificationRequestOptions', $notificationRequestOptions);
 
+		// If a user is also assigned as an author to this submission, they
+		// shouldn't see any editorial actions
+		if ($stageId === WORKFLOW_STAGE_ID_INTERNAL_REVIEW || $stageId === WORKFLOW_STAGE_ID_EXTERNAL_REVIEW) {
```

Comment:
`$stageId` will always be `WORKFLOW_STAGE_ID_INTERNAL_REVIEW` or `WORKFLOW_STAGE_ID_EXTERNAL_REVIEW` in this code, no? (Or is `$stageId` the submission's "current" stage, which might be later?)

Reply:
That seems likely but I wasn't entirely sure, so I checked anyway. On looking further, I can see that it doesn't really matter for this check, so I'll remove it.